data/mods/gen1stadium/scripts.ts

```diff
@@ -26,8 +26,8 @@ export const Scripts: ModdedBattleScriptsData = {
 				this.baseStoredStats[statName] = this.storedStats[statName] = Math.floor(stat);
 				this.modifiedStats![statName] = Math.floor(stat);
 				// Re-apply drops, if necessary.
-				if (this.status === 'par') this.modifyStat!('spe', 0.25);
-				if (this.status === 'brn') this.modifyStat!('atk', 0.5);
+				if (this.status === 'par' && statName === 'spe') this.modifyStat!('spe', 0.25);
+				if (this.status === 'brn' && statName === 'atk') this.modifyStat!('atk', 0.5);
```

Comment:
Both of these are unintentional, but Paralysis was accidentally working correctly, thanks to a quirk where Speed is the last stat, and `modifiedStats` is reset right before the second-to-last quartering. Still, this code is much more readably correct.